### PR TITLE
skill frontmatter の不変条件を機械検証 (description 1024 字制限の実障害由来)

### PR DESCRIPTION
**Pull request by kanade0404**
## Summary
- **実障害由来**: codex exec が `.agents/skills/issue-driven-development/SKILL.md` の読み込みに失敗 (`invalid description: exceeds maximum length of 1024 characters`)。description が 1093 字で Anthropic 公式制約 (≤1024) を超過していた。Claude Code は寛容に読むため顕在化せず、**他エンジンとの相互運用で初めて露見**
- issue-driven-development の description を 1024 字以内に圧縮 (トリガ語彙・negative space は維持、trigger 結果 JSONL を再測定して同梱)
- **sensor 昇格**: `tests/test_skill_frontmatter.py` — 全 skill の frontmatter 不変条件を機械検証
  - description ≤1024 字 (今回の障害クラス)
  - name = ディレクトリ名 (CLAUDE.md の配布不変条件を初めて機械化)
  - name の文字種・長さ (公式制約)
- rulesync mirror 再生成済み (drift check 通過)

## 検証
- unittest: 全 skill を subTest で検証し OK (現カタログで違反は修正した 1 件のみだったことを事前計測済み)
- score_triggers.py: 新結果 JSONL は 13/13 一致
- `node scripts/rulesync-sync.mjs --check`: ✓ / ruff: clean

## Test plan
- [x] ローカル unittest / scorer / drift check
- [ ] CI (trigger-evals は PR #50 merge 後は unittest ステップでこのテストも毎回実行される)

🤖 Generated with [Claude Code](https://claude.com/claude-code)

https://claude.ai/code/session_01XcnmrjKiVvAYmDbCcixzX3

## Summary by CodeRabbit

* **Documentation**
  * Updated the issue-driven development skill guidance for clearer, shorter workflow instructions and handling of edge cases.
  * Clarified when automation stops and when follow-up actions are handled separately.

* **Tests**
  * Added automated checks to verify skill metadata consistency across all skill files.
  * Added evaluation data covering expected and out-of-scope trigger scenarios.

### CODE REVIEW

File: tests/test_skill_frontmatter.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+def extract_description(frontmatter: str) -> str | None:
+    match = DESCRIPTION_RE.search(frontmatter)
+    if match is None:
+        return None
+    if match.group("block") is not None:
+        lines = []
+        for line in match.group("block").splitlines():
+            lines.append(line[2:] if line.startswith("  ") else line)
+        return "\n".join(lines).strip()
```
[the remark this answers is not in the document]
Comment (kanade0404) [reply tests/test_skill_frontmatter.py:43]: Fixed in f0446bb (follow-up PR #53) — インデント幅を最初の内容行から決定するようにし、2 space 固定の仮定を除去しました (4 space のテストケース追加)。

File: tests/test_skill_frontmatter.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
+"""skills/*/SKILL.md の frontmatter 不変条件を機械検証する (stdlib unittest).
```
[the remark this answers is not in the document]
Comment (kanade0404) [reply tests/test_skill_frontmatter.py:1]: Fixed in f0446bb (follow-up PR #53) — tests/__init__.py を追加し、素の python3 -m unittest discover でも発見されることを確認済みです (31 tests OK)。

File: tests/test_skill_frontmatter.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (kanade0404) [reply tests/test_skill_frontmatter.py:43]: Fixed in f0446bb (follow-up PR #53) — chomping を考慮し、| (clip) は末尾改行 1 つを長さに数える実装に変更しました (|- は数えない。単体テスト付き)。

**Comment by kanade0404** [comment]
## Review Response Summary (post-merge)
merge 後に残っていた 3 スレッドはすべて VALID と判定し、follow-up PR #53 (f0446bb) で対応して resolve しました: インデント幅の動的決定 / unittest discover の発見性 (tests/__init__.py) / chomping を考慮した長さ判定。
